libs/core/src/lib/feed-input/feed-input.component.scss

```diff
@@ -4,6 +4,28 @@
     &.is-disabled {
         pointer-events: initial;
         opacity: initial;
+
+        textarea {
+            &::placeholder {

```

Comment:
You also need `color: #32363a` for IE11

Reply:
@InnaAtanasova could you please re-review? Did I understand all of your comments right?

Reply:
Yes! Thank you for fixing it. We need this extra line of css code to handle IE11.